cloud/docker_client.go

```
@@ -266,6 +268,46 @@ func (c *dockerClientImpl) ListContainers(ctx context.Context, h *host.Host) ([]
 	return containers, nil
 }
 
+// ListImages lists all images on the specified host machine.
+func (c *dockerClientImpl) ListImages(ctx context.Context, h *host.Host) ([]types.ImageSummary, error) {
+	dockerClient, err := c.generateClient(h)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to generate docker client")
+	}
+
+	// Get all container images
+	opts := types.ImageListOptions{All: false}
+	images, err := dockerClient.ImageList(ctx, opts)
+	if err != nil {
+		err = errors.Wrap(err, "Docker list API call failed")
+		grip.Error(err)
```

You don't need to both log and return the error - I would just return the error